molecule/zabbix_proxy/molecule.yml

```diff
@@ -19,9 +19,13 @@ platforms:
     volumes:
       - /sys/fs/cgroup:/sys/fs/cgroup:rw
     cgroupns_mode: host
+    zabbix_proxy_version: ${MY_MOLECULE_VERSION:-v70}
```

Comment:
zabbix_proxy_version is assigned when the instance is a part of the v70 group down below, i think.

Reply:
No, below variables participate when we converge. I need this variable at the container creation time and for some reason it's not available at this time (probably because the host (container) is still not added to the inventory Molecule is using later to converge.

Reply:
Aha, yes i've spotted `image: "zabbix/zabbix-server-pgsql:ubuntu-{{ p_to_s[item.zabbix_proxy_version] }}-latest"`. That's actually quite clever just using the official zabbix images for the tests :+1:

Reply:
Yep :+1: